src/common/token/claims.go

```diff
@@ -0,0 +1,30 @@
+package token
+
+import (
+	"github.com/dgrijalva/jwt-go"
+	"github.com/goharbor/harbor/src/common/rbac"
+	"github.com/pkg/errors"
+)
+
+// RobotClaims implements the interface of jwt.Claims
+type RobotClaims struct {
+	jwt.StandardClaims
+	TokenID   int64          `json:"id"`
```

Comment:
I think we need to revisit the structure of the token to make it more extensible, but this should not block this PR.